src/jobs/cleanup.js

```diff
@@ -85,10 +146,14 @@ export default function (agenda) {
 
 			await processBlobs({
 				client, processCallback, messageCallback,
-				query: {state: [BLOB_STATE.PROCESSED, BLOB_STATE.ABORTED]},
+				query: {state},
 				filter: blob => {
 					const modificationTime = moment(blob.modificationTime);
-					return modificationTime.add(ttl).isBefore(moment());
+					if (before) {
```

Comment:
No need to introduce a new option because the `method`-option can be used here as well: `method === 'reQueueBlob`